core/extension/extension_api_dump.cpp

```diff
@@ -994,6 +998,9 @@ Dictionary GDExtensionAPIDump::generate_extension_api() {
 					if (index != -1) {
 						d2["index"] = index;
 					}
+
+					d2["usage"] = F.usage;
```

Comment:
```suggestion
					if (F.usage) {
						d2["usage"] = F.usage;
					}
```
Let's not store if it is zero, no need to take up that space, granted a large portion of properties have non-zero usage but still

Reply:
Fair enough I'll add it when I wake up in the morning